### PR TITLE
Remove base docker image

### PR DESCRIPTION
**Pull request by hlts2**
### Description:

I deleted the base docker image and dependencies.

### Environment:

- Go Version: 1.15.7
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.13.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [x] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [x] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dockers/dev/Dockerfile [hunk 1]
```diff
@@ -14,14 +14,19 @@
...
+FROM ubuntu:devel AS builder
 ARG PROTOBUF_VERSION=3.14.0
-ARG GOLANGCI_LINT_VERSION=v1.35.2
+ARG GOLANGCI_LINT_VERSION=v1.36.0
 ARG REVIEWDOG_VERSION=v0.11.0
-ARG YQ_VERSION=4.3.2
+ARG YQ_VERSION=4.5.0
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
```
Comment (rinx) [inline dockers/dev/Dockerfile:26]:
```suggestion
    ca-certificates \
    curl \
```
https://github.com/vdaas/vald/pull/995/checks?check_run_id=1861034779

**Review by rinx — COMMENTED** [review]
Sorry, maybe `go.mod` and `go.sum` are missing in dockers/ci/base/Dockerfile
https://github.com/vdaas/vald/pull/995/checks?check_run_id=1861034923

File: dockers/dev/Dockerfile (hunk 1 above)
Comment (hlts2) [reply dockers/dev/Dockerfile:26]: thank you for your comment :bow:
I will fix it soon :bow:

**Review by rinx — APPROVED** [review]

**Comment by rinx** [comment]
Thanks, great job :+1:

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**